utils.js

```diff
@@ -102,15 +102,18 @@ async function checkResultAndReturnUrl(
 
 export async function openBrowserAsync(
   url: string,
-  options?: InAppBrowserOptions = {
+  options?: InAppBrowserOptions = {}
+): Promise<BrowserResult> {
+  const defaults = {
```

Comment:
Hello mate, thanks for your contribution!
Can you move the default settings outside this method please? and can you include the type `InAppBrowserOptions` for that new variable?